Clarify whether router.pathname includes basePath (#24675)
vercel/next.js: docs/api-reference/next/router.md

The following is the definition of the `router` object returned by both [`useRouter`](#useRouter) and [`withRouter`](#withRouter):

> - `pathname`: `String` - Current route. That is the path of the page in `/pages`, and it does not include the configured `basePath`.

Reviewer: ```suggestion
- `pathname`: `String` - Current route. That is the path of the page in `/pages`, the configured `basePath` or `locale` is not included.
```
